tests/test_reportportal.py

```
@@ -405,6 +460,27 @@ def _make_failure(self, classification="PRODUCT BUG", details="Analysis text"):
         failure.analysis.code_fix = None
         return failure
 
+    def _setup_push_client(self, *, locators=None, put_side_effect=None):
+        """Create a ReportPortalClient with mocked session for push tests."""
+        client = ReportPortalClient(
+            url="http://rp.example.com", token="tok", project="proj"
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "token"

(S106)